index.bs

```diff
@@ -4908,6 +4981,20 @@ a numbered step. If outdented, it (today) is rendered as a bullet in the midst o
                   [=[RP]=]-specific. </dd>
            </dl>
 
+1. If <code>|response|.{{AuthenticatorAssertionResponse/attestationObject}}</code> is present and the [=[RP]=] wishes to verify the attestation then perform CBOR decoding on {AuthenticatorAssertionResponse/attestationObject}} to obtain the attestation statement format |fmt|, and the attestation statement |attStmt|.
+
+    1. Verify that the `AT` bit in the [=flags=] field of |authData| is set, indicating that [=attested credential data=] is included.
+
+    1. Verify that [=credentialId=] and [=credentialPublicKey=] fields of the [=attested credential data=] in |authData| match the stored values for the selected [=credential public key=].
+
+    1. Determine the attestation statement format by performing a USASCII case-sensitive match on |fmt| against the set of supported WebAuthn Attestation Statement Format Identifier values. An up-to-date list of registered WebAuthn Attestation Statement Format Identifier values is maintained in the IANA "WebAuthn Attestation Statement Format Identifiers" registry [[!IANA-WebAuthn-Registries]] established by [[!RFC8809]].
+
+    1. Verify that |attStmt| is a correct [=attestation statement=], conveying a valid [=attestation signature=], by using the [=attestation statement format=] |fmt|'s [=verification procedure=] given |attStmt|, |authData| and |hash|.
+
+        Note: Each [=attestation statement format=] specifies its own [=verification procedure=]. See [[#sctn-defined-attestation-formats]] for the initially-defined formats, and [[!IANA-WebAuthn-Registries]] for the up-to-date list.
+
+    1. If validation is successful, obtain a list of acceptable trust anchors (i.e. attestation root certificates) for that attestation type and attestation statement format |fmt|, from a trusted source or from policy. The [=aaguid=] in the [=attested credential data=] can be used to guide this lookup.
+
```

Comment:
What should an RP do if it requested attestation during an assertion ceremony, and it is *not* present in the response? This use case is omitted from the processing rules, but it is not clear what that means to an RP?

Reply:
I think that during registration, the output of the attestation needs to be stored and associated with the credential, so that during an authentication (assertion), a decision can be made by consulting the attestation response agains the known credential attestation and if that is valid and should be present. So this will need to likely indicate that the attestation of the credential and it's details are stored with the credential so that this decision can be made (webauthn-rs already does this, because it allows post-registration verification of credential properties).